src/go/migration-cli/tests/data/complex_cluster_001_output.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  annotations:
+    cluster.redpanda.com/managed: "false"
+  creationTimestamp: null
+  name: complex-cluster-001
+spec:
+  chartRef:
+    chartVersion: 4.0.54
+  clusterSpec:
+    auth:
+      sasl:
+        enabled: true
+        users: null
+    console:
+      enabled: false
+    external:
+      domain: someblah.byoc.prd.cloud.redpanda.com
+    image:
+      repository: docker.io/redpandadata/redpanda
+      tag: v23.1.11
+    license_secret_ref:
+      secret_key: license
+      secret_name: redpanda-someblah-license
+    listeners:
+      admin:
+        external: {}
+        port: 9644
+        tls:
+          cert: ""
+          enabled: false
+          requireClientAuth: false
+      http:
+        authenticationMethod: none
+        enabled: true
+        external:
+          http-default:
+            advertisedPorts: []
```

Comment:
Why this is present?

Reply:
do you mean why we define an empty list?

Reply:
Why migration tool produces empty advertised ports list.

Reply:
what should it be?